.github/workflows/dist.yml

```diff
@@ -0,0 +1,83 @@
+name: Dist
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+
+concurrency:
+  group: dist-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build_wheels:
+    name: "Build Wheels ${{ matrix.buildplat }}"
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+      matrix:
+        buildplat: ["win_amd64", "win32"]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build wheels
+        uses: pypa/cibuildwheel@v2.17.0
+        env:
+          CIBW_BUILD: "cp3*-${{ matrix.buildplat }}"
+          CIBW_PRERELEASE_PYTHONS: "True"
+          CIBW_TEST_COMMAND: "python -c \"import winkerberos;print(winkerberos.__version__)\""
+
+      - name: Assert all versions in wheelhouse
+        if: ${{ ! startsWith(matrix.buildplat[1], 'macos') }}
+        run: |
+          ls wheelhouse/*cp37*.whl
```

Comment:
We've deprecated 3.7, do we still need this?

Reply:
I'll do a separate PR to drop 3.7